### PR TITLE
Make container-apigentools read image version from env or from config.json

### PR DESCRIPTION
**Pull request by bkabrda**
### What does this PR do?

This makes `container-apigentools` get the image to use from environment or from configuration file. While doing this change, I rewrote `container-apigentools` to Python to be able to better implement this new feature. I also updated the docs to promote usage of the containerized version.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Files changes must correspond to the primary purpose of the PR as described in the title (small unrelated changes should have their own PR)
- [ ] Feature or bugfix must have tests
- [ ] Git history [must be clean](https://github.com/DataDog/apigentools/blob/master/CONTRIBUTING.md#commit-messages)

### CODE REVIEW

File: apigentools/container_cli.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            command.append("-e")
+            command.append("{}={}".format(k, v))
+    for mountdir, mountpoint in mountpoints.items():
+        command.append("-v")
+        command.append("{}:{}".format(mountdir, mountpoint))
+    command.append(image)
+    command.extend(args.apigentools_args)
+
+    # if using latest, explictly pull to actually get latest image
+    if image.endswith(":latest"):
```
Comment (jirikuncar) [inline apigentools/container_cli.py]:
```suggestion
    if ":" not in image or image.endswith(":latest"):
```

Comment (bkabrda) [reply apigentools/container_cli.py]: Good point!

**Comment by bkabrda** [comment]
@zippolyte @jirikuncar thanks for your reviews! I accepted the suggestion from @jirikuncar and also added one minor commit with a message that will inform users when they try to run new `container-apigentools` the old way. Please re-review.

**Comment by bkabrda** [comment]
Thanks for your reviews! Merging.

**Pull request merged**